src/query/settings/src/lib.rs

```diff
@@ -388,6 +388,13 @@ impl Settings {
                 desc: "the max number of rows each read from parquet to databend processor",
                 possible_values: None,
             },
+            SettingValue {
+                default_value: UserSettingValue::UInt64(0),
+                user_setting: UserSetting::create("max_result_rows", UserSettingValue::UInt64(0)),
+                level: ScopeLevel::Session,
+                desc: "Auto limit max result rows if user not specify the limit, default is 0 means no limit",
```

Comment:
`Auto limit max result rows if user not specify the limit, default is 0 means no limit`-->`Auto limit max result rows if user specify the limit, default is 0 means no limit`?

Reply:
`max_result_rows` does't take affect  If user specifies the limit ,such as select * from xx limit 10.

Reply:
Ok.
It would be better to add a new case to the test: `select count() from (select * from t1 limit 2)` after settings `SET max_result_rows=1`.

Reply:
ok